Paintroid/src/main/java/org/catrobat/paintroid/MainActivity.java

```diff
@@ -436,15 +436,20 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 	}
 
 	@Override
-	public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
-		presenter.handlePermissionRequestResults(requestCode, permissions, grantResults);
+	public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+		presenter.handleRequestPermissionResult(requestCode, permissions, grantResults);
 	}
 
 	@Override
-	public void forwardActivityResult(int requestCode, int resultCode, Intent data) {
+	public void baseActivityResult(int requestCode, int resultCode, Intent data) {
```

Comment:
since this is called from the presenter whenever you want to have the super class handle the result, what do you think about calling this `superOnActivityResult(..)` or something similar that contains the term `super`?

Reply:
Thanks, changed to `super.*`